src/vivarium_public_health/risks/implementations/low_birth_weight_and_short_gestation.py

```diff
@@ -336,7 +336,7 @@ def setup(self, builder: Builder) -> None:
 
         super().setup(builder)
         self.interpolator = self.get_interpolator(builder)
-        self.relative_risk = self.get_relative_risk_pipeline(builder)
+        self.relative_risk = self.get_relative_risk(builder)
```

Comment:
You should be able to remove this line entirely if you take my suggestions in `RiskEffect`.